scipy/linalg/decomp.py

```diff
@@ -953,14 +953,14 @@ def eigvalsh(a, b=None, lower=True, overwrite_a=False,
         *Deprecated by ``driver=gvd`` option*. Has no significant effect for
         eigenvalue computations since no eigenvectors are requested.
 
-        ..Deprecated in v1.5.0
+        .. Deprecated:: 1.5.0
```

Comment:
capitalization here is different from below--seems to be a complaint from the CI:

`sphinx.errors.SphinxParallelError: KeyError: 'Deprecated'`

Reply:
Dang; fixed.